generated_types/regenerate-flatbuffers.sh

```diff
@@ -24,45 +24,78 @@ FB_COMMIT="86401e078d0746d2381735415f8c2dfe849f3f52"
 # - Check in the changes to the generated code along with your changes to the `Cargo.lock` file and
 #   this script.
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-pushd $DIR
+# By default, this script will run a Docker container that uses the same image we use in CI that
+# will have all the necessary dependencies. If you don't want to run Docker, run this script with
+# LOCAL=1.
+if [ -z ${LOCAL} ]; then
+  echo "Running in Docker..."
 
-echo "Building flatc from source ..."
+  CI_IMAGE=quay.io/influxdb/rust:ci
 
-FB_URL="https://github.com/google/flatbuffers"
-FB_DIR=".flatbuffers"
-FLATC="$FB_DIR/bazel-bin/flatc"
+  DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+  pushd $DIR
 
-if [ -z $(which bazel) ]; then
-    echo "bazel is required to build flatc"
-    exit 1
-fi
+  DOCKER_IOX_DIR=/home/rust/influxdb_iox
+
+  docker kill flatc || true
+  docker rm flatc || true
+
+  docker pull ${CI_IMAGE} || true
 
-echo "Bazel version: $(bazel version | head -1 | awk -F':' '{print $2}')"
+  docker run \
+    -it \
+    --detach \
+    --name=flatc \
+    --volume ${DIR}/..:${DOCKER_IOX_DIR} \
+    ${CI_IMAGE}
 
-if [ ! -e $FB_DIR ]; then
-    echo "git clone $FB_URL ..."
-    git clone -b master --no-tag $FB_URL $FB_DIR
+  docker exec -e LOCAL=1 flatc .${DOCKER_IOX_DIR}/generated_types/regenerate-flatbuffers.sh
+
+  docker kill flatc || true
+  docker rm flatc || true
```

Comment:
You could mirror the Rust code with

```suggestion
  docker rm --force flatc || true
```